Python/optimizer_bytecodes.c

```diff
@@ -949,7 +970,7 @@ dummy_func(void) {
        next = sym_new_type(ctx, &PyLong_Type);
     }
 
-    op(_CALL_TYPE_1, (unused, unused, arg -- res)) {
+    op(_CALL_TYPE_1, (callable, null, arg -- res, c, a)) {
         PyObject* type = (PyObject *)sym_get_type(arg);
         if (type) {
             res = sym_new_const(ctx, type);
```

Comment:
You need to replace the constant propagation code here, it's no longer `_POP_CALL_ONE_LOAD_CONST_INLINE_BORROW` !

Reply:
Ah yeah that explains the failure. I was actually wondering if I need to update this optimization. Thanks!

Reply:
Close! You need this uop to now match the new `_CALL_TYPE_1` which is `(unused, unused, arg -- res, a)`. Change it to a uop that swaps things around:
Instead of `op(_POP_CALL_ONE_LOAD_CONST_INLINE_BORROW, (ptr/4, callable, null, pop -- value)`, make it
`_SWAP_CALL_ONE_LOAD_CONST_INLINE_BORROW, (ptr/4, callable, null, arg -- value, a)`, the `_POP_TOP` that comes after it will automatically get specialized.

Reply:
Yes indeed. I can work on making the optimizer append rather than do things in-place, but I don't want to go through the massive rounds of review that will take, so I'm putting it off for now.

Reply:
and I just realized this silently breaks all the `_CALL_TYPE_1` tests. I also need to teach `remove_unneeded_uops` about `_SWAP_CALL_ONE_LOAD_CONST_INLINE_BORROW`.

Reply:
Ugh that's really bad. I forgot about that. Thank you.